PocketKit/Sources/PocketKit/Tags/TagsFilterViewModel.swift

```diff
@@ -6,7 +6,11 @@ import Textile
 import SharedPocketKit
 
 class TagsFilterViewModel: ObservableObject {
-    private var fetchedTags: [Tag]?
+
+    /// Grab the latest tags from the database on each ask for them to ensure we are up to date
+    private var fetchedTags: [Tag] {
+        self.source.fetchAllTags() ?? []
```

Comment:
seems like we're getting the data via source here and then using a different way of getting the data via the source's viewContext in the view. Is that okay as opposed to one source of truth?

Reply:
@cyndichin this only exists for RecentTags

Reply:
as well as checking if we should delete and rename tags it seems; not a blocker though just a thought